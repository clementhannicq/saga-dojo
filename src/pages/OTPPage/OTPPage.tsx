import React, { SFC } from "react";

const OTPPage: SFC<{}> = () => <p>OTP Page here</p>;

export default OTPPage as any;
