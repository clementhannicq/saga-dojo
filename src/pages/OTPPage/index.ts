import OTPPage from "./OTPPage";

export default OTPPage;
