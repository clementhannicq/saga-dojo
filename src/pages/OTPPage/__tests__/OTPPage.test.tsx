import { shallow } from "enzyme";
import React from "react";
import OTPPage from "../OTPPage";

describe("Pages", () => {
  describe("OTPPage", () => {
    it("should render", () => {
      const wrapper = shallow(<OTPPage />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
