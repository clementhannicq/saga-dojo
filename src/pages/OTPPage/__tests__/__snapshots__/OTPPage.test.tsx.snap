// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages OTPPage should render 1`] = `
<p>
  OTP Page here
</p>
`;
